repository-data/development/src/main/resources/hcm-config/main.yaml

```diff
@@ -4,7 +4,8 @@ definitions:
     /hippo:configuration/hippo:modules/autoexport/hippo:moduleconfig:
       autoexport:enabled: false
       autoexport:modules:
-        operation: add
-        type: string
-        value:
-        - repository-data/development
+      - repository-data/development:/
```

Comment:
Adding auto export modules to allow hst config to be exported correctly.
Currently, hst configuration is not exported locally when developing.

Reply:
I worry about it changing the workflow.